vgen/src/main/resources/io/gapi/vgen/py/discovery_fragment.snip

```diff
@@ -0,0 +1,187 @@
+@snippet generateFilename(method)
+    {@method.getName}.frag.py
+@end
+
+@snippet generateModule(method, body, imports)
+    @let api = context.getApi, \
+         apiName = api.getName, \
+         apiVersion = api.getVersion, \
+         serviceTitle = context.getService.getTitle
+        """
+        PRE-REQUISITES:
+        ---------------
+        1. If not already done, enable the {@serviceTitle} and check quota for your project at
+           https://console.developers.google.com/apis/api/{@apiName}_component/quotas 
+        2. Install python client library for Google APIs using 'pip install --upgrade google-api-python-client'
+        3. This sample uses Application Default Credentials for Auth. If not already done, install the gcloud CLI from 
+           https://cloud.google.com/sdk/ and run 'gcloud beta auth application-default login'
+        """
+    
+        @join import : imports
+            {@import}
+        @end
+        
+        @# Authentication is provided by gcloud tool when running locally
```

Comment:
s/by/by the/
s/gcloud/"gcloud"/  (check whether we need to do this for other languages)